### DISCUSSION TITLE
"Version History" button cannot be localized

### DISCUSSION THREAD

**Discussion by isametry** [MonitorControl/MonitorControl#1078] [General]
### Describe the bug

The string for the "Version History" button shown is excluded from all localization files, and thus always stays in English:

![Snímek obrazovky 2022-06-05 v 14 32 35](https://user-images.githubusercontent.com/46103746/172050869-0e900787-ab12-4e8f-a352-62842a4be092.png)

### Steps to reproduce

`Menu → Check for Updates…`

or alternatively:

`Menu → Preferences → About → Check for updates…`

### Expected behavior

The "Version History" button string should be localized, according to the espective lproj.

### Environment Information

```markdown
- macOS version: macOS 12.4
- Mac model: MacBook, Late 2020
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): —
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
The reason for this is that it is part of Sparkle which is a third party component. You might want to contribute to that project: https://github.com/sparkle-project/Sparkle

If they release an update, it will eventually find it's way to MonitorControl. :)

**Comment by Jerry23011** [comment]
I think it's included in Sparkle's latest version, please see [this file](https://github.com/sparkle-project/Sparkle/blob/2.x/Sparkle/zh_CN.lproj/Sparkle.strings), at line 92
